pkg/sync/runtime_spinning_amd64.s

```diff
@@ -14,7 +14,7 @@
 
 // https://go.dev/cl/669235 (1.25) adds a new schedt field prior to nmspinning.
 // https://go.dev/cl/714800 (1.26) changes size of the schedt.midle field.
```

Comment:
I think for consistency both these comment lines should probably be removed

Reply:
PTALA